pkg/config/worklist.go

```diff
@@ -0,0 +1,39 @@
+package config
+
+import (
+	"github.com/bipol/scrapedumper/pkg/martaapi"
+	"github.com/bipol/scrapedumper/pkg/worker"
+	"go.uber.org/zap"
+)
+
+//WorkConfig is the top-level config object, defined an entire
+//scrapedumper job to be started.
+type WorkConfig struct {
+	BusDumper   DumpConfig `json:"bus_dumper"`
+	TrainDumper DumpConfig `json:"train_dumper"`
+}
+
+//BuildWorkList builds a worklist from the specified clients
+//and dumper config
+func BuildWorkList(
```

Comment:
q: i'm curious how this will be built out if we start to make "scrapers" a little more generic, and less marta centric.  Not that you should change your pr at all, but might be a good design discussion to have in the future to make this a little more extensible!

Reply:
Yes definitely! I think including scrapers in the config would imply changing the structure of this thing in a breaking way, but I figured that would be an acceptable compromise since we have a pretty limited deployment right now (I said a thing or two about that up in the PR description). I imagine a list of scraper/dumper pairs instead?